lib/ride.rb

```diff
@@ -0,0 +1,29 @@
+require 'pry'
+class Ride
+    attr_reader :name, :min_height, :admission, :excitement, :rider_log
+
+    def initialize(info)
+        @name = info[:name]
+        @min_height = info[:min_height]
+        @admission = info[:admission]
+        @excitement = info[:excitement]
+        @rider_log = {}
+    end
+
+    def board_rider(visitor)
+        return unless can_board?(visitor)
+        @rider_log[visitor] ||= 0
+        @rider_log[visitor] += 1
+        visitor.take_money(@admission)
+    end
+
+    def can_board?(visitor)
```

Comment:
Nice job making this helper method. Makes the board_rider method very easy to read.